Makefile

```
@@ -315,7 +315,10 @@ lint-deps: ## Validate architecture/import boundaries with dependency-cruiser
 	node scripts/generateLocalization.mjs
 	$(PM_EXEC) $(DEPCRUISE_BIN) src pages tests --config .dependency-cruiser.js
 
-lint: lint-next lint-tsc lint-md lint-deps ## Runs all linters: ESLint, TypeScript, Markdown, and dependency-cruiser in sequence.
+lint-headers: ## Verify the edge security-header policy (config/security-headers.json) reaches every production response
+	$(PM_EXEC) node scripts/ci/lint-headers.mjs
+
+lint: lint-next lint-tsc lint-md lint-deps lint-headers ## Runs all linters: ESLint, TypeScript, Markdown, dependency-cruiser, and the security-header gate in sequence.
```

checkmake (0.3.2)

[warning] Target "lint" should be declared PHONY.

(phonydeclared)